src/main/java/org/folio/dew/controller/BulkEditController.java

```diff
@@ -234,7 +245,20 @@ public ResponseEntity<String> uploadCsvFile(UUID jobId, MultipartFile file) {
         var job = getBulkEditJob(jobCommand);
         var jobLaunchRequest = new JobLaunchRequest(job, jobCommand.getJobParameters());
         log.info("Launching bulk edit user identifiers job.");
-        exportJobManagerSync.launchJob(jobLaunchRequest);
+        var tenantId = folioExecutionContext.getTenantId();
+        var headers = folioExecutionContext.getOkapiHeaders();
+        new Thread(() -> {
+          Map<String, Collection<String>> okapiHeaders = new HashMap<>(headers);
+          okapiHeaders.put("x-okapi-tenant", List.of(tenantId));
```

Comment:
As far as I understand, there is no need to put tenantId header as it is already there.

Reply:
I would prefer leave it as it is since it has already been tested with this line of code.